components/odh-notebook-controller/controllers/notebook_controller_test.go

```diff
@@ -740,7 +740,7 @@ func createOAuthContainer(name, namespace string) corev1.Container {
 	return corev1.Container{
 		Name:            "oauth-proxy",
 		Image:           OAuthProxyImage,
-		ImagePullPolicy: corev1.PullAlways,
+		ImagePullPolicy: corev1.PullIfNotPresent,
```

Comment:
previously, test did not run thru because it was still assuming imagePullPolicy Always in IS vs MUST BE comparision during testing